modules/utils.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable */
+export let scores = [];
+/* eslint-enable */
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :smile: